code/Cargo.toml

```diff
@@ -93,8 +95,8 @@ frame-system = { git = "https://github.com/ComposableFi/substrate", branch = "po
 frame-system-benchmarking = { git = "https://github.com/ComposableFi/substrate", branch = "polkadot-v0.9.36" }
 frame-system-rpc-runtime-api = { git = "https://github.com/ComposableFi/substrate", branch = "polkadot-v0.9.36" }
 frame-try-runtime = { git = "https://github.com/ComposableFi/substrate", branch = "polkadot-v0.9.36" }
-pallet-assets = { git = "https://github.com/ComposableFi/substrate", branch = "polkadot-v0.9.36" }
-pallet-asset-tx-payment = { git = "https://github.com/ComposableFi/substrate", branch = "polkadot-v0.9.36" }
+# pallet-assets = { git = "https://github.com/ComposableFi/substrate", branch = "polkadot-v0.9.36" }
```

Comment:
how come is this being commented?

Reply:
i have remomoved unused dependency. ci linter failed with patching unused. i have commemted it out so when it will become used(when xcm tests enabled back).